### PR TITLE
add updateInternship endpoint

### PR DESCRIPTION
**Pull request by ShirshovAlex**
add update methop to internship

### CODE REVIEW

**Review by Maksim-maevsky — DISMISSED** [review]

File: src/main/java/by/exadel/internship/mapper/internship/InternshipMapper.java [hunk 1]
```diff
@@ -42,4 +39,8 @@
     @IterableMapping(qualifiedByName = "internship")
     List<UserInternshipDTO> mapUserInternshipDTOList(List<Internship> internshipList);
 
+    @BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE,
```
Comment (kzaretskaya) [inline src/main/java/by/exadel/internship/mapper/internship/InternshipMapper.java]: Why do you ignore null value for mapping?

File: src/main/java/by/exadel/internship/controller/InternshipController.java [hunk 2]
```diff
@@ -87,4 +87,11 @@ public void addUser(@RequestParam(name = "userId") UUID userId, @PathVariable(na
     public List<Skill> getSkillList (){
         return Arrays.asList(Skill.values());
     }
+
+    @SuperAdminAccessControl
+    @PutMapping("/{internshipId}/update")
```
Comment (kzaretskaya) [inline src/main/java/by/exadel/internship/controller/InternshipController.java]: Why do you need `update` word?
```suggestion
    @PutMapping("/{internshipId}")
```

File: src/main/java/by/exadel/internship/mapper/internship/InternshipMapper.java (hunk 1 above)
Comment (ShirshovAlex) [reply src/main/java/by/exadel/internship/mapper/internship/InternshipMapper.java]: remove it

File: src/main/java/by/exadel/internship/controller/InternshipController.java (hunk 2 above)
Comment (ShirshovAlex) [reply src/main/java/by/exadel/internship/controller/InternshipController.java]: done

**Review by kzaretskaya — APPROVED** [review]

**Review by Maksim-maevsky — APPROVED** [review]

**Pull request merged**